src/globals/helpers/_media-queries.scss

```diff
@@ -11,6 +11,6 @@ $sass-mq-already-included: false !default;
   $mq-show-breakpoints: ();
 }
 
-@import "./node_modules/sass-mq/mq";
+@import "../tools/sass-mq";
```

Comment:
It seems unnecessary to split media queries across these two layers… and because sass-mq 'outputs' CSS (even if it is just breakpoints) it would make more sense in here. Would suggest just moving sass-mq in here and using `exports` to simplify the hack as @nickcolley identified.